specs/commandline/verify.md

```diff
@@ -37,14 +52,16 @@ Usage:
 Flags:
        --allow-referrers-api         [Experimental] use the Referrers API to verify signatures, if not supported (returns 404), fallback to the Referrers tag schema
   -d,  --debug                       debug mode
+       --file                        enable verifying a file, if set, the reference argument is the file path or full URI reference of the file artifact in registry (required if --signature is set)
   -h,  --help                        help for verify
        --insecure-registry           use HTTP protocol while connecting to registries. Should be used only for testing
        --max-signatures int          maximum number of signatures to evaluate or examine (default 100)
        --oci-layout                  [Experimental] verify the artifact stored as OCI image layout
   -p,  --password string             password for registry operations (default to $NOTATION_PASSWORD if not specified)
        --plugin-config stringArray   {key}={value} pairs that are passed as it is to a plugin, if the verification is associated with a verification plugin, refer plugin documentation to set appropriate values
-       --scope string                [Experimental] set trust policy scope for artifact verification, required and can only be used when flag "--oci-layout" is set
+       --scope string                [Experimental] set trust policy scope for artifact verification, required when flag "--oci-layout" is set, can only be used when "--oci-layout" or "--file" is set
   -u,  --username string             username for registry operations (default to $NOTATION_USERNAME if not specified)
+       --signature stringArray       path of signatures when verifying a file, required and used if and only if the target file is stored in file system
```

Comment:
What will be output if user passed multiple signatures? Also, how can user consume that programmatically ?

Reply:
It's very similar to our current behavior. Currently, if there are multiple signatures attached to the target artifact in the registry, Notation would verify those signatures one by one.
For verifying a file, signatures are still verified one by one. Again, example on multiple signatures is provided in this PR. Please take a look there.

Reply:
That's a good point. Should we still follow the current behavior though? In other words, we still claim that as long as 1 signature passed the verification, the whole verification succeeded. @yizha1 @FeynmanZhou Do you have any comment on this one?

Reply:
I think this will require more discussion and to start with we can support only one signature.

Reply:
I see. Then as a start, we will make `--signature` a string type instead of stringArray.